Vpn.Service/Downloader.cs

```diff
@@ -60,15 +60,13 @@ public async Task ValidateAsync(string path, CancellationToken ct = default)
 
         if (fileSigInfo.State != SignatureState.SignedAndTrusted)
             throw new Exception(
-                $"File is not signed and trusted with an Authenticode signature: State={fileSigInfo.State}");
+                $"File is not signed and trusted with an Authenticode signature: State={fileSigInfo.State}, StateReason={fileSigInfo.StateReason}");
 
         // Coder will only use embedded signatures because we are downloading
         // individual binaries and not installers which can ship catalog files.
         if (fileSigInfo.Kind != SignatureKind.Embedded)
             throw new Exception($"File is not signed with an embedded Authenticode signature: Kind={fileSigInfo.Kind}");
 
-        // TODO: check that it's an extended validation certificate
```

Comment:
So, I looked into this, and we can check for extended validation. It's a little involved, so let's leave this as a TODO. I'll open a GitHub issue.